lib/rules/prefer-numeric-literals.js

```diff
@@ -6,13 +6,42 @@
 "use strict";
 
 //------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+/**
+ * Checks to see if a CallExpression's callee node is `parseInt` or
+ * `Number.parseInt`.
+ * @param {ASTNode} calleeNode The callee node to evaluate.
+ * @returns {boolean} True if the callee is `parseInt` or `Number.parseInt`,
+ * false otherwise.
+ */
+function isParseInt(calleeNode) {
+    if (calleeNode) {
```

Comment:
I personally don't like having unused checks "just in case". If a `CallExpression` doesn't have a `callee` for whatever reason, some invariant has been violated and the rule is likely to do the wrong thing anyway, since we don't know what the "right thing" would be.  This also makes it much harder to detect bugs in other parts of the code (in this case, the parser) because the rule just fails silently rather than throwing if it encounters something unexpected.